adapter/service/outlvlswitch/routing.go

```diff
@@ -168,3 +242,29 @@ func HandleCmdLvlGetReport(serviceRegistry adapter.ServiceRegistry) router.Messa
 		}),
 	)
 }
+
+func getStartLvl(message *fimpgo.Message) (int, error) {
+	switch d, ok, err := message.Payload.Properties.GetIntValue(StartLvl); {
+	case !ok:
+		log.Info("adapter: start_lvl not found in message properties")
```

Comment:
Do not log if this optional property is not provided. It's normal case, it does not require logging.